src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -157,8 +156,7 @@ public function getDebug()
     public function getFlashes($types = null)
     {
         try {
-            $session = $this->getSession();
-            if (null === $session) {
+            if (null === $session = $this->getSession();
```

Comment:
`if (null === $session = $this->getSession()) {`

Reply:
fixed thanks